gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/InitialImageOperation.java

```diff
@@ -232,6 +232,16 @@ private GIIStatus reportGIIStatus() {
     }
   }
 
+  private boolean hasLockedBucket = false;
+
+  private void unlockTargetBucketLock(InternalDistributedMember recipient) {
+    if (region instanceof BucketRegion && hasLockedBucket) {
+      this.region.getDistributionManager().getLoggerI18n().info(LocalizedStrings.DEBUG, "Processing SnapshotBucketLockReleaseProcessor to "+ recipient);
+      SnapshotBucketLockReleaseMessage.send(recipient, this.region.getDistributionManager(), region.getFullPath());
```

Comment:
Yes I am more worried about the target node if exception comes while sending or receiving the response.